examples/Dockerfile.web

```diff
@@ -22,21 +16,16 @@ RUN npm ci
 
 # Development stage installs devDependencies, builds app from source and declares a file watcher as the default command.
 #   We name this stage so we can refer to it in later stages
-FROM $REGISTRY/ffc-node-development:$DEV_VERSION AS development
+FROM $REGISTRY/ffc-node-development:$BASE_VERSION AS development
 
 # Expose the PORT passed in to the Dockerfile, and also some development debugging ports that are used during development
 EXPOSE ${PORT} 9229 9230
 
-# As this is a new container, we want to ensure that we set up the working directory and user correctly
-USER node
-WORKDIR /home/node
-
 # We copy the installed packages from the production "base" install that was the first stage.
 COPY --from=base --chown=node:node /home/node/package*.json ./
-COPY --from=base --chown=node:node /home/node/node_modules ./node_modules
 
-# We run a full dev install to bring in any development packages that the build might require to run
-RUN npm install
+# We run a full dev install to bring in any development packages
+RUN npm install --production=false

```

Comment:
I can't see any mention of `--production=false` in the docs. I suspect this will have no effect.